cinema/views.py

```diff
@@ -167,6 +186,25 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "list", "item": {"type": "date"}},
```

Comment:
We search by one date, not period, nor several dates. type = {"type" : "string", "format" : "date"} would've been better option.

Reply:
thank you, this is a good comment, thank you for the example, I will do better